src/common/fullstory.ts

```diff
@@ -9,27 +7,36 @@ import * as FullStory from '@fullstory/browser';
  * and set the flag to true if the number is below the sampling percentage.
  */
 
-const FULLSTORY_RECORD_KEY = 'FULLSTORY_RECORD';
-
 // Percentage of overall users that we want to record
 const FULLSTORY_RECORD_PERCENT = 1 / 100;
 
+const FULLSTORY_RECORD_KEY = 'FULLSTORY_RECORD';
+
 enum RecordSession {
   TRUE = 'true',
   FALSE = 'false',
 }
 
-export function initFullStory() {
+function isRecordingEnabled() {
   if (!storageAvailable('localStorage')) {
-    return;
+    return false;
   }
-
   initStorage();
+  return localStorage.getItem(FULLSTORY_RECORD_KEY) === RecordSession.TRUE;
+}
 
-  // If we can't find the value in localStorage we won't enable recordings in FullStory
-  if (localStorage.getItem(FULLSTORY_RECORD_KEY) === RecordSession.TRUE) {
-    FullStory.init({ orgId: 'XEVN9' });
-  }
+async function getFullStory() {
+  return isRecordingEnabled() ? import('@fullstory/browser') : null;
+}
+
+export async function initFullStory() {
+  return getFullStory().then(fs => (fs ? fs.init({ orgId: 'XEVN9' }) : null));
+}
```

Comment:
nit: I think these functions would be more readable if they returned `void` and used `await` and `if` like so:

```
export async function initFullStory(): void {
  const fs = await getFullStory();
  if (fs) {
    fs.init({ orgId: 'XEVN9' });
  }
}
```

If you wanted to avoid boilerplate and make a helper, you could do:

```
// TODO: Use proper type instead of any.
export async function withFullStory(fn: (fs: any) => void): void {
  const fs = await getFullStory();
  if (fs) {
    fn(fs);
  }
}
...
withFullStory(fs => fs.init({ordId: 'KEVN9'}));
```